sections/all_map_locations.tex

```diff
@@ -115,15 +120,12 @@ \subsection*{Symbols on the Map}
 
   \phantom{j}\textbf{Pandora's Box}\\
   \framedimage[\linewidth]{\map_locations/pandoras_box.jpg}\\
-}
 
-\vspace*{\fill}
-
-\vspace{7.4em}
+  \bigskip
 
-\note{10}{
-  The effects of Fields that allow you to \textbf{Search} for Spells or Artifacts, or where you must spend resources to use the Field's effect, are \textbf{always optional}.
-  You must always place a Black Cube on such Visitable Fields even if you choose not to use that Field's effect.
+  \phantom{j}\textbf{Windmill}\\
```

Comment:
It's still off by some pixels, I'm not sure why.

Reply:
I see it, but no idea why...

Reply:
Please experiment. I put in a lot of effort to make everything match perfectly.

Reply:
For some reason windmill's frame is rendered taller by 1 pixel